project_resource_calendar/models/calendar_event.py

```diff
@@ -49,6 +49,11 @@ class CalendarEvent(models.Model):
         string='Is Created From Task',
         default=False,
     )
```

Comment:
Ras Partner Sector should go in project_resource_calendar. Otherwise there is a loop dependency

Reply:
@eilst  Done